apps/web/components/booking/TimeOptions.tsx

```diff
@@ -15,8 +13,7 @@ type Props = {
 
 const TimeOptions: FC<Props> = ({ onSelectTimeZone }) => {
   const [selectedTimeZone, setSelectedTimeZone] = useState("");
-  const query = useMeQuery();
-  const userTheme = useTheme(query?.data?.theme).resolvedTheme;
+  const userTheme = useTheme().resolvedTheme;
```

Comment:
TimeOptions is used on booking page and booking page theming is based on the organizer being booked and not the logged in user.